app/1.0/configuration.md

```diff
@@ -311,10 +311,10 @@ and Kong will be running in that instance (as configured in `nginx-kong.conf`).
 
 ## Serving both a website and your APIs from Kong
 
-A common use case for API providers is to make Kong serve both a website
+A common use case for Service providers is to make Kong serve both a website
 and the APIs themselves over the Proxy port &mdash; `80` or `443` in
-production. For example, `https://my-api.com` (Website) and
-`https://my-api.com/api/v1` (API).
+production. For example, `https://example.com` (Website) and
+`https://example.com/api/v1` (Service).
```

Comment:
Same, this `Service` should probably be `API` for clarity

Reply:
Undid this one too.